src/vs/workbench/browser/parts/editor/tabsTitleControl.ts

```diff
@@ -337,8 +339,25 @@ export class TabsTitleControl extends TitleControl {
 				}
 			}
 
-			// Figure out scrolling direction
-			const nextEditor = this.group.getEditorByIndex(this.group.getIndexOfEditor(activeEditor) + (e.deltaX < 0 || e.deltaY < 0 /* scrolling up */ ? -1 : 1));
+			// Ignore event if the last one happened too recently.
+			// The restriction is relaxed according to the abs value of deltaX and deltaY
+			const time_threshold = 150;
```

Comment:
I suggest to extract this to a `static` constant below [here](https://github.com/microsoft/vscode/blob/f9f2637f794fb728b32d39c25e9eb8a5763afe1d/src/vs/workbench/browser/parts/editor/tabsTitleControl.ts#L74-L74)